### PR TITLE
Feature/spaces

### PR DESCRIPTION
**Pull request by blusol850**

### CODE REVIEW

File: models/AmazonS3.cfc [hunk 1]
```diff
@@ -54,23 +54,30 @@ component accessors="true" singleton {
      */
     public AmazonS3 function init(
         required string accessKey,
-        required string secretKey,
+		required string secretKey,
```
Comment (lmajano) [inline models/AmazonS3.cfc:59]: Can you add the arguments to the CFC docs please. Also they should be proper properties as well

File: models/AmazonS3.cfc [hunk 2]
```diff
@@ -54,23 +54,30 @@ component accessors="true" singleton {
...
+
         setSSL( ssl );
 
         variables.sv4Util = new Sv4Util(
             accessKeyId = variables.accessKey,
             secretAccessKey = variables.secretKey,
             defaultRegionName = arguments.awsRegion,
-            defaultServiceName = "s3"
-        );
+            defaultServiceName = 's3'
```
Comment (lmajano) [inline models/AmazonS3.cfc:76]: Where is the change to this util. I don’t see it?

Comment (byandrewdavis) [reply models/AmazonS3.cfc]: Just single quotes. Probably shouldn't have been pushed.

Comment (blusol850) [reply models/AmazonS3.cfc:76]: it was just single quotes around s3, probably should have been excluded from commit

File: models/AmazonS3.cfc (hunk 1 above)
Comment (blusol850) [reply models/AmazonS3.cfc:59]: javadoc notation added. 
Explain what you mean about proper properties please.

Comment (lmajano) [reply models/AmazonS3.cfc:59]: Adding them as `property name="secretKey"`

Comment (lmajano) [reply models/AmazonS3.cfc:59]: Basically adding them to the pseudo constructor so they are documented with auto getters and setters:

```
property name="awsRegion";
property name="awsDomain";
```

File: models/AmazonS3.cfc (hunk 2 above)
Comment (lmajano) [reply models/AmazonS3.cfc:76]: NO, what I mean, is that the default Service Name argument is not in the sv4Util code? Where is it?

**Review by lmajano — CHANGES_REQUESTED** [review]

**Pull request merged**